XIPs/xip-53-v2-deprecation.md

```diff
@@ -0,0 +1,76 @@
+---
+xip: 53
+title: XMTP V2 deprecation
+description: This XIP defines a plan to turn off writes to the XMTP V2 network and making it read only to encourage developers to upgrade to using the XMTP V3 network.
+author: Naomi Plasterer (@nplasterer)
+discussions-to: TBD
+status: Draft
+type: Standards
+category: XRC
+created: 2025-01-29
+---
+
+## Abstract
+
+XMTP V3 provides the framework for a more robust messaging ecosystem—supporting stronger security guarantees with MLS, modern features like group chats, and decentralization. These enhancements weren’t efficiently and effectively feasible with XMTP V2, which also can’t migrate seamlessly to V3.
+
+To encourage adoption of XMTP V3 and ensure the network remains secure and future-proof, we provide this plan to remove the ability to write to XMTP V2. This step discourages reliance on outdated infrastructure and fosters a unified community transition to the more advanced V3 network.
+
+## Motivation
+
```

Comment:
Also took great feedback from @cameronvoell here as well - and rewrote a bit. WDYT?